libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationConfig.kt

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.navigation.dropin
+
+import android.content.Context
+import com.mapbox.navigation.dropin.ViewOptionsCustomization.Companion.defaultRouteArrowOptions
+import com.mapbox.navigation.dropin.ViewOptionsCustomization.Companion.defaultRouteLineOptions
+import com.mapbox.navigation.ui.maps.NavigationStyles
+import com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions
+import com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+/**
+ * DropInNavigationView configuration holder class.
+ */
+internal class NavigationConfig(context: Context) {
```

Comment:
Not 100% set on name here. Any suggestions?

Reply:
Let's delete the existing `NavigationViewOptions` and use the same name.